unetsocket/c/Makefile

```diff
@@ -1,14 +1,18 @@
 CC = gcc
 CFLAGS += -std=c99 -Wall -Wextra -Werror -Wfloat-equal -Wconversion -Wparentheses -pedantic -Wunused-parameter -Wunused-variable -Wreturn-type -Wno-unused-function -Wredundant-decls -Wreturn-type -Wunused-value -Wswitch-default -Wuninitialized -Winit-self -O2
-# FJAGE_VER=c8e62495f2283d1b57a559112ccce3ccd608bb2a
 FJAGE_VER=master
```

Comment:
fjage's `master` was broken a while ago for compiling `fjage.c`. We should verify if this works on modems.

Reply:
I have been testing with the latest release of fjage from `master`. It is working fine with the modems running the latest unet v3.3.0.